app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt

```diff
@@ -70,7 +74,7 @@ class CreditCardEditorFragment : SecureFragment(R.layout.fragment_credit_card_ed
                 args.creditCard?.toCreditCardEditorState(storage)
                     ?: getInitialCreditCardEditorState()
             }
-            creditCardEditorView = CreditCardEditorView(binding, interactor)
+            creditCardEditorView = CreditCardEditorView(binding, interactor, ::showDeleteDialog)
```

Comment:
Let's add `showDeleteDialog` into `DefaultCreditCardEditorController` instead. I think we prefer to keep all the logic inside of the controller instead. Please also add an @param when doing the refactor. I think the change will be simply enough that it does not require another review.